analysis/generate_report.py

```diff
@@ -159,6 +161,12 @@ def generate_report(experiment_names,
 
     data_utils.validate_data(experiment_df)
 
+    # Loads the json summary file.
```

Comment:
nit: `Load` not `Loads`. The style is a bit confusing, but it seems like descriptive style (ends in "s", see [this](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)) is used for docstrings, while we generally don't do this for regular comments.